src/redux/slices/books/bookSlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+// const myBooks = [
+//     {
+//       title: 'Fantastic Beasts And Where To find them',
+//       author: 'J. K. Rowling',
+//       category: 'action',
+//       chapter: 'Chapter 17',
+//     },
+//     {
+//       title: 'The Hunger games',
+//       author: 'Suzanne Collins',
+//       category: 'action',
+//       chapter: 'Chapter 3',
+//     },
+//     {
+//       title: 'Jurassic Parc',
+//       author: 'Michael Crichton',
+//       category: 'sci-fi',
+//       chapter: 'Introduction',
+//     },
```

Comment:
- [ ] I suggest that you remove the commented code in your code base. Leaving commented-out code in source code is bad practice, as it takes up space, causes confusion, and leads to maintenance issues if it's not removed 😞.